src/styles/quiz-card/quiz-card.css

```
@@ -0,0 +1,47 @@
+.quiz__general {
+  background: var(--background);
+  border-color: var(--color-white);
+  color: var(--color-blue);
+  display: grid;
+  grid-gap: 12px;
+  max-width: 600px;
+  padding: 10px;
+  position: relative;
+}
+
+.quiz-card__heading {
+  font-size: 1.2rem;
+  font-weight: 400;
+  text-align: center;
+}
+
+.quiz-cart__button {
+  background: var(--color-primary);
+  color: var(--color-white);
+}
+
+.quiz-card__taglist {
+  padding: 0;
+  column-gap: 20px;
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  place-content: center;
+}
+
+.quiz-card_answer {
+  visibility: hidden;
+}
+
+.quiz-card__bookmark {
+  border: 10px solid white;
+  border-bottom-color: transparent;
+  height: 30px;
+  position: absolute;
+  right: 10px;
+  top: -10px;
+}
+
+.quiz-card__bookmark--added {
+  border: 10px solid black;
```

I think you'll need to add this line as well for it to work properly. 
```suggestion
  border: 10px solid black;
  border-bottom-color: transparent;
```